dll/win32/kernel32/kernel32_vista/vista.c

```diff
@@ -19,6 +19,43 @@
 #define NDEBUG
 #include <debug.h>
 
+/* INTERNAL FUNCTIONS *********************************************************/
+
+static BOOL DevicePathToDosPath(LPWSTR lpPath, DWORD dwSize)
```

Comment:
To implement the fix, use something like:
```suggestion
static BOOL
DevicePathToDosPath(
    _Out_ PWSTR pDosPath,
    _In_ DWORD dwPathSize,
    _In_ PCWCH pDevicePath,
    _In_ DWORD dwDevicePathSize)
```

Reply:
Done.